src/app/shared/constants.ts

```diff
@@ -26,14 +26,44 @@ export const KIV_ZONES_MAP = KIV_ZONES.reduce((acc, zone) => {
   return acc;
 }, {});
 
+export enum VOLUNTEER_STATUS {}
+
 export enum VOLUNTEER_ROLES {
-  delivery = 'delivery',
+  delivery = 'Livrare',
+  copilot = 'Copilot',
+  packing = 'Împachetare',
+  supply = 'Aprovizionare',
+  operator = 'Operator',
+}
+
+export enum VOLUNTEER_ROLES_ICONS {
+  delivery = 'transport',
   copilot = 'copilot',
-  packing = 'packing',
-  supply = 'supply',
+  packing = 'impachetare',
+  supply = 'aprovizionare',
   operator = 'operator',
 }
 
+export enum ZONES {
```

Comment:
Is this a duplicate of `KIV_ZONES` above?

Reply:
Yes, back-end uses constants in enum format. So the global approach looks like Front-End and Back-End should use the same defined constants. Theoretically frontEnd may import such constants via a API call. That's why we tried to align constants usage to the Back-End approach.

Reply:
as discussed and accepted by phone - this enum comply with back-end.